build/common.props

```diff
@@ -4,7 +4,7 @@
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
     <MinorVersion>$(MinorVersionPrefix)34</MinorVersion>
-    <PatchVersion>0</PatchVersion>
+    <PatchVersion>2</PatchVersion>
```

Comment:
I already used patch version 1 (but from a different release branch https://github.com/Azure/azure-functions-host/commits/release/in-proc8/ ) and published 4.834.1 last week.

I will delete that release branch to avoid confusion.